### DISCUSSION TITLE
X-Button

### DISCUSSION THREAD

**Discussion by RundesBalli** [community/community#13143] [Other Feature Feedback, Questions, & Ideas]
I would like to have a button to mark recommendations as "I never want to see **this exact** suggestion again".

For example: I don't like or use vue.js, and I want to click on a "x-Button" in the top right corner of the recommendation, to hide this recommendation for ever, because I don't want to waste my time looking at a repository with vue.js in it.

Same for the "Explore" feed. There are repositories that I've seen 5,000 times in the Explore feed, but I can't mark as "I don't care".

**Comment by jfullstackdev** [comment]
fortunately, they are working on it, 

I'm just commenting to update this post

and 

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
